CONTRIBUTING.rst

```diff
@@ -262,6 +262,19 @@ Here is a simple overview, with pytest-specific bits:
 
    When committing, ``pre-commit`` will re-format the files if necessary.
 
+#. If instead of using ``tox`` you prefer to run the tests directly, then we suggest to create a virtual environment and use
+   an editable install with the ``testing`` extra::
+
+       $ python3 -m venv .env
```

Comment:
s/.env/.venv/

`.env` is often used for envdirs etc, and `.venv` is more common from what I can tell.